scripts/bitwarden.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+set -e
+
+cat << EOF
+ _     _ _                         _
+| |__ (_) |___      ____ _ _ __ __| | ___ _ __
+| '_ \| | __\ \ /\ / / _` | '__/ _` |/ _ \ '_ \
+| |_) | | |_ \ V  V / (_| | | | (_| |  __/ | | |
+|_.__/|_|\__| \_/\_/ \__,_|_|  \__,_|\___|_| |_|
+
+EOF
+
+cat << EOF
+Open source password management solutions
+Copyright 2015-$(date +'%Y'), 8bit Solutions LLC
```

Comment:
If this file came from the upstream Bitwarden repos, then this would probably come with an (A)GPL license which is not acceptable for this project.